python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -910,6 +913,29 @@ def _create_table_optimize_block(self, pserver_index, pserver_program,
 
         return table_opt_block
 
+    def _create_checkpoint_save_block(self, pserver_program, pre_block_idx):
+        """
+        create a new block to handle save checkpoint.
+        """
+        import os
+
+        pserver_program.global_block().create_var(
+            name="lookup_table_path",
+            persistable=True,
+            type=core.VarDesc.VarType.RAW)
+
+        checkpoint_save_block = pserver_program.create_block(pre_block_idx)
+        checkpoint_save_block.append_op(
+            type='save',
+            inputs={'X': [self.table_name]},
+            outputs={},
+            attrs={
+                'file_path':
```

Comment:
```python
'file_path': ""
```
put `this 'file_path' do not be used in save lookup table variable` into comment

Reply:
done